tests/src/test/common.cpp

```diff
@@ -1,7 +1,5 @@
 #include "test/common.h"
 
-#include <gtest/gtest.h>
-
```

Comment:
What's going on here? We use `gtest.h` for `::testing::AddGlobalTestEnvironment` and `::testing::Environment`. Removing this include triggers clang-tidy.